authentication.md

```diff
@@ -77,10 +77,17 @@ When a user is successfully authenticated, they will be redirected to the `/home
 
 If the redirect path needs custom generation logic you may define a `redirectTo` method instead of a `redirectTo` property:
 
-    protected function redirectTo()
-    {
-        //
-    }
+        /**
+         * Generate post-authentication redirect location
+         * 
+         * @since 5.3.29 Introduced redirectTo function support in 5.3.29 
```

Comment:
can you elaborate?  looked right to me based on the docs page https://phpdoc.org/docs/latest/references/phpdoc/tags/since.html